demo/EXPLOR3R/auto-drive.py

```diff
@@ -99,7 +101,7 @@ def turn():
     t = randint(250, 750)
 
     for m, p in zip(motors, power):
-        m.run_timed(duty_cycle_sp=p*75, time_sp=t)
+        m.run_timed(speed_sp=p*750, time_sp=t)
```

Comment:
Good catch; I guess we didn't fix all cases of `duty_cycle` in the demos after we made that change in the library.